release-team/role-handbooks/emeritus-adviser/README.md

```diff
@@ -0,0 +1,72 @@
+# Emeritus Adviser Handbook
+
+## Overview
+
+The Emeritus Adviser (EA) is an experienced Release Team (RT) member who returns to the team in an advisory role.
+
+## Responsibilities
+
+The primary duty of the EA is
+managing personnel for the release team, including Role Leads and especially Shadows.  The EA monitors the Shadow program and making sure that it is working as an effective apprenticeship process.
+
+As a returning RT member, the EA is also there to supply the Release Lead with historical information about how things were handled in prior releases, and serve as a sounding board when both the Release Lead and Lead Shadows are facing a dilemma.
+
+## Requirements
+
+The EA is a returning senior Release Team member, ideally a former Release Lead.  If not a former Lead, the EA should either have served in more than one Role on the Release Team, or be a lead in SIG-Release, or both.  They must be tempermentally suited to people management, including a commitment to diversity and helping others ascend the Contributor Ladder.
+
+Time commitments for the EA are relatively light, with the largest time commitment in the two-week period where Shadows are being selected.  However, the EA role starts 2-3 weeks before the release cycle officially begins, and continues for 1-2 weeks after it ends.
+
+Due to timing (see below) EAs are not chosen as part of the regular RT selection process.  Instead, interested candidates should contact the outgoing EA and the incoming Release Lead 2 weeks before the final prior release.  Most of the time, the outgoing EA will solicit suitable candidates.
+
+Due to the nature of the role, the EA has no shadows.  The job may, however, be split between more than one person.
+
+## Tasks
+
+### Release Team Selection
+
+Weeks: -2 to 0
+
+At least 2 weeks before the new release cycle begins, the EA is responsible for posting the "Assemble the X.Y Release Team" issue in the SIG-Release repo.  They should then make sure that the new Release Lead is officially nominated and chosen, and then issue a call for nominations/volunteers for the other Role Leads.
```

Comment:
This would be the outgoing EA doing this vs the incoming EA correct?

Reply:
No, it's the incoming EA.

Reply:
I did it this time around because we didn't have a process yet. But the EA for X.Y should own the whole process from week -2 to week 14.